go/signedexchange/structuredheader/parser_test.go

```diff
@@ -0,0 +1,260 @@
+package structuredheader
```

Comment:
Let's make it a different package `structured_header`. We'd need to export some functions.

Reply:
I thought this is the preferred way?

https://golang.org/pkg/testing/#pkg-overview

> To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the “go test” command is run. For more detail, run “go help test” and “go help testflag”.

Reply:
We discussed offline and concluded it would not be worth making this a different package by exposing a lot of functions. (There is a technique to expose functions only for testing, but it is a little messy). Let's keep as this is. Thanks.